### PR TITLE
Display New Version information from changelog at bottom of Info Page

### LINKED ISSUE #174: Include a way to view the changelog in Aspine
It would be nice if users could see (in a non-intrusive way) what new features have been added to Aspine. We could do this by rendering `CHANGELOG.md` to HTML in the backend and serving that via an endpoint which is loaded upon a button click in the info tab (after #157 is merged; that blocker is why this issue is currently on hold). In the lite version build process, we could include a step that runs a script in JS that does this compilation (we might do well to rewrite the build script entirely in JS in that case).

### PR DESCRIPTION
**Pull request by kdk1616**
* Parses Markdown from CHANGELOG.md and displays at bottom of info page

### CODE REVIEW

**Review by tektaxi — APPROVED** [review]
Looks good, glad to be able to see updates in Aspine now.

File: public/home.html
Comment (psvenk) [inline public/home.html]: This CSS rule should be moved to `home.css`.

File: public/home.html
Comment (psvenk) [inline public/home.html]: - It seems that you have an unclosed `<center>` tag
- [The `<center>` tag is deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center); you should assign an ID to the `<a>` and use CSS instead.

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: I don't think we should have this much whitespace. I would just remove all whitespace between the end of the `for` loop adn teh end of the callback function and leave only one line between the end of the entire AJAX call and the `//#endif` (for some reason GitHub isn't letting me put this as a "suggestion" with a diff).

File: serve.js
Comment (psvenk) [inline serve.js]: As before, I would also remove this whitespace, especially because you did not touch this code and it would be good to have a cleaner overall diff.

File: serve.js
Comment (psvenk) [inline serve.js]: Instead of using type coercion (adding a string to `changelog` to convert it from a `Buffer` to a string), use [`Buffer.prototype.toString`](https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end):
```suggestion
    res.send(marked(changelog.toString()));
```

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: Use `const` instead of `var` and [`document.querySelectorAll`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll) instead of `getElementById` and `getElementsByTagName`:
```suggestion
    const items = document.querySelectorAll("#updates h2");
```

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: The indentation for this entire callback should be normalized.

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: Instead of using a C-style `for` loop prone to off-by-one errors, this can be done more simply:
```suggestion
	items.forEach(x => { x.className = "info-header"; });
```

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: This can also be done more directly and in a way that avoids potential conflict with other styling (just good practice in general):
```suggestion
    items.slice(3).forEach(x => { x.style.setProperty("display", "none") });
```

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: This can also be simplified:
```suggestion
    const subpoints = document.querySelectorAll("#updates ul");
    subpoints.slice(3).forEach(x => { x.style.setProperty("display", "none") });
```

**Review by psvenk — CHANGES_REQUESTED** [review]
Looks good overall (from reading the diff), though I have not yet tested your code. Here are some code style changes (but take them with a grain of salt because I haven't tested any of my proposed changes).

**Comment by notrodes** [comment]
Stop writing jquery!!

**Comment by notrodes** [comment]
Im fine not rewriting stuff, just stop writing morehow hard is that!?!?!?

**Review by notrodes — DISMISSED** [review]
delete $

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: @kdk1616 Why did you change `const` to `var` here? We should always avoid `var` in favor of `const` or `let`.

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: Remove log statement
```suggestion
```

File: public/js/home.js
Comment (psvenk) [inline public/js/home.js]: Remove log statement
```suggestion
```

**Review by psvenk — CHANGES_REQUESTED** [review]

File: public/css/home.css
Comment (psvenk) [inline public/css/home.css]: Use kebab-case for CSS IDs and classes, not camelCase
```suggestion
#more-versions {
```

File: public/home.html
Comment (psvenk) [inline public/home.html]: Remove extraneous space between `a` and `href` and prevent overlong lines; use
kebab-case instead of camelCase for CSS ID
```suggestion
      <p id="more-versions">
        <a href="https://github.com/Aspine/aspine/blob/master/CHANGELOG.md">
          &lt&lt See Older Versions of Aspine &gt&gt
        </a>
      </p>
```

**Review by psvenk — CHANGES_REQUESTED** [review]

**Review by psvenk — APPROVED** [review]
Looks good to me; I think this is ready to merge pending @notrodes' approval. I noticed that the changelog inclusion does not work in Aspine Lite, but I chose to split that into a separate issue (#250) (a) this is a sufficiently small feature, (b) the link to GitHub still exists on Aspine Lite, and (c) implementing this would require considerable work on the build script.

**Pull request merged**
